project_forecast_line_deadline/tests/test_forecast_line_deadline.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as common
+
+
+class TestForecastLineDeadline(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.project_task = self.env.ref("project.project_task_1")
+
+    def test_forecast_end_deadline(self):
+        """Test forecast_date_planned_end vs date_deadline"""
+        task = self.project_task
+        # See that relevant date fields are falsy
+        self.assertFalse(task.forecast_date_planned_end)
+        self.assertFalse(task.date_deadline)
+        # Set deadline for task
+        task.date_deadline = "2022-01-01"
+        # Date fields are equal
+        self.assertEqual(task.forecast_date_planned_end, task.date_deadline)
+        # Set forecast_date_deadline manually
+        task.forecast_date_planned_end = "2022-02-01"
```

Comment:
Ah, maybe this answers my question - writing to the field does not write None, but it writes the value that was actually written?

In that case another question - isn't that the way computed fields with `readonly=False` normally behave? And if so, why don't we use these? Or do they exist only from Odoo 15.0?

Reply:
Basically whenever I wanted to make a computed field writable too, I set an inverse that does nothing actually. This makes the field not to be read-only. I haven't tried setting readonly=False in such case, at least it did not work for versions <14. Will check it out

Reply:
I've quickly scanned the Odoo sourcecode and it seems the `readonly=False` construct is already used in Odoo 14.0. Maybe you can check if it does the same.

eg: https://github.com/OCA/OCB/blob/14.0/addons/hr_holidays/models/hr_leave.py#L131